src/api/readCpmProduct/src/v1/steps.py

```diff
@@ -0,0 +1,60 @@
+from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
+from domain.repository.cpm_product_repository.v3 import CpmProductRepository
+from domain.response.validation_errors import mark_validation_errors_as_inbound
+from event.step_chain import StepChain
+
+
+@mark_validation_errors_as_inbound
+def _parse_event_path_parameters(path_parameters: dict):
+    """
+    Parse path parameters to extract the required IDs (product_id and product_team_id).
+    """
+    product_id = path_parameters.get("product_id")
+    product_team_id = path_parameters.get("product_team_id")
+
+    if not product_id:
+        raise ValueError("Missing required path parameter: 'product_id'")
+
+    if not product_team_id:
+        raise ValueError("Missing required path parameter: 'product_team_id'")
+
+    return product_id, product_team_id
```

Comment:
Better putting this in a pydantic model like the query params (but instead path params), that way these error messages can be generated by default

Reply:
ah yeah and then could return the model rather than the dict

Reply:
Where does the query params do this? The search params?